www/src/styles/global.css

```diff
@@ -159,26 +159,23 @@
   }
 
   .markdown .remark-code-title {
-    @apply z-10 -mb-1 mt-3 w-full rounded-t-md border-b-2 border-t3-purple-200 bg-[#292D3E] px-2 py-1 font-mono text-t3-purple-200;
+    @apply z-10 -mb-1 mt-3 w-full rounded-t-md border border-t3-purple-300/80 bg-[#0F111A] px-2 py-1 font-mono text-t3-purple-200;
   }
 
   .markdown pre {
-    @apply mx-auto mb-3 rounded-md p-2 pl-3 font-mono sm:pt-2;
+    @apply mx-auto mb-3 rounded border border-t3-purple-300/80 shadow-xl p-2 pl-3 font-mono sm:pt-2;
   }
 
   .markdown ol {
     @apply w-full;
   }
 
   .markdown details {
-    @apply mt-2 w-full rounded-md  bg-[#292D3E] px-3 transition-all duration-75 ease-in-out;
+    @apply mt-2 w-full rounded-md  bg-[#0F111A] px-3 transition-all duration-75 ease-in-out;
```

Comment:
These look a bit weird with the border on the inner element?

![image](https://user-images.githubusercontent.com/51714798/201797955-b047b39c-18d0-4128-9964-2d5759bb7825.png)

Reply:
yep, even I don;t like it in the details block, is there any way we can filter it out from details block

Reply:
ig we can by using css, something like `.details > pre` maybe